### PR TITLE
[14.0][FIX] stock_request: action_view_stock_requests

### PR DESCRIPTION
**Pull request by Kosaaaaa**
FIX action_view_stock_requests for non admin.

Users without admin rules could not open this action.

### CODE REVIEW

**Review by sebalix — APPROVED** [review]
LG

File: stock_request/__manifest__.py [hunk 1]
```diff
@@ -4,12 +4,13 @@
 {
     "name": "Stock Request",
     "summary": "Internal request for stock",
-    "version": "14.0.1.0.4",
+    "version": "14.0.1.0.5",
```
Comment (etobella) [inline stock_request/__manifest__.py]: No need to change the version. Will be done by runbot

File: stock_request/models/stock_request_order.py [hunk 2]
```diff
@@ -259,7 +259,7 @@ def action_view_transfer(self):
         return action
 
     def action_view_stock_requests(self):
-        action = self.env.ref("stock_request.action_stock_request_form").read()[0]
+        action = self.env.ref("stock_request.action_stock_request_form").sudo().read()[0]
```
Comment (rousseldenis) [inline stock_request/models/stock_request_order.py]:
```suggestion
        action = self.env["ir.actions.act_window"]._for_xml_id("stock_request.action_stock_request_form")
```

File: stock_request/__manifest__.py (hunk 1 above)
Comment (rousseldenis) [inline stock_request/__manifest__.py]: You don't need to update this unless you have no migrations script in your PR. OCA git bot will update it automatically.

**Review by rousseldenis — CHANGES_REQUESTED** [review]

**Comment by rousseldenis** [comment]
@Kosaaaaa Do you plan to fix this ?

**Comment by Kosaaaaa** [comment]
> @Kosaaaaa Do you plan to fix this ?

Fixed

**Comment by rousseldenis** [comment]
@Kosaaaaa Could you squash your commits as soon as tests are done ?

**Comment by Kosaaaaa** [comment]
> @Kosaaaaa Could you squash your commits as soon as tests are done ?

No problem

**Comment by Kosaaaaa** [comment]
> @Kosaaaaa Could you squash your commits as soon as tests are done ?

done

**Comment by rousseldenis** [comment]
Seems that this one passed before. Sorry
https://github.com/OCA/stock-logistics-warehouse/pull/1326

**Pull request closed without merging**